Modules/Quotas/src/Quotas.Application/Infrastructure/Persistence/Repository/IIdentitiesRepository.cs

```diff
@@ -7,5 +7,7 @@ public interface IIdentitiesRepository
     Task Add(Identity identity, CancellationToken cancellationToken);
     Task<IEnumerable<Identity>> FindWithTier(TierId tierId, CancellationToken cancellationToken, bool track = false);
     Task Update(IEnumerable<Identity> identities, CancellationToken cancellationToken);
+    Task Update(Identity identity, CancellationToken cancellationToken);
+    Task<Identity> FindByAddress(string address, CancellationToken cancellationToken, bool track = false);
```

Comment:
Can simply be called `Find`. because the Address in case of an Identity is like the Id of any other entity. And don't call it `FindById` either.